app/v2/notifications/post_notifications.py

```diff
@@ -686,7 +685,7 @@ def get_reply_to_text(notification_type, form, template, form_field=None):
 
 
 def strip_keys_from_personalisation_if_send_attach(personalisation):
-    return {k: v for (k, v) in personalisation.items() if not (type(v) is dict and v.get("sending_method") == "attach")}
+    return {k: v for (k, v) in personalisation.items() if not (isinstance(v, dict) and v.get("sending_method") == "attach")}
```

Comment:
ruff changes this automatically? 👀

Reply:
This one was a manual change, this check [(E721)](https://docs.astral.sh/ruff/rules/#E721) is not automagically fixable.